template/android/fastlane/Fastfile

```diff
@@ -31,7 +31,11 @@ platform :android do
     gradle(
       task: "assemble",
       flavor: "#{env.capitalize}",
-      build_type: "Release"
+      build_type: "Release",
+      properties: {
+        "VERSION_NAME" => version,
```

Comment:
This make us have 2 places to config `VERSION_NAME`
We already have 1 `VERSION_NAME` defined in `android/app/build.gradle` and `android/gradle.properties`
Should we combine these values?

Reply:
the `VERSION_NAME` you se in `android/gradle.properties` is the define for verions_name when you build in local it will get the default. when the pipeline run it will get from package.json and set to `VERSION_NAME`